libs/blocks/bulk/utils.js

```diff
@@ -0,0 +1,269 @@
+import { getImsToken } from '../../../tools/send-to-caas/send-utils.js';
+import { fetchWithTimeout } from '../caas/utils.js';
```

Comment:
This seems suboptimal to reuse a utility from another block. Instead, the common bits should be moved to a more central location IMO.

Reply:
@honstar I addressed your comment as follows:
- I moved the bulk publish code below milo/tools to make it a tool (like floodgate), not a block
- I moved the methods used from specific blocks/tools to some shared utils
See commit https://github.com/adobecom/milo/pull/559/commits/e3ee3cb327d5098c5f6dd1f18491484ff2e18c58